src/utils/puns.ts

```diff
@@ -56,9 +83,22 @@ const ERROR_PUNS = [
   'You are battering FSH',
   'Swim back to School of FSH',
   'You need to mullet over',
```

Comment:
I just noticed we also have `You should mullet over` in the warnings section. Maybe remove this one?

Reply:
Removed.